test/spec_lint.rb

```diff
@@ -742,14 +742,6 @@ def each
                      }).call(env("rack.input" => eof_weirdio))
     }.must_raise(Rack::Lint::LintError).
       message.must_match(/read\(nil\) returned nil on EOF/)
-
```

Comment:
This spec should be switched to check for a successful result, not removed.

Reply:
Updated, thanks!